docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+        from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+        from aws_lambda_powertools.logging import correlation_paths
+
+        logger = Logger(service="APP")
+
+        app = ApiGatewayResolver()
+
+
+        cold_start = True
+
+
+        @app.get("/hello/<name>")
+        def hello_name(name):
+            with xray_recorder.in_subsegment("hello_name") as subsegment:
+                logger.info(f"Request from {name} received")
+                subsegment.put_annotation("User", name)
+                return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+        @app.get("/hello")
+        def hello():
+            with xray_recorder.in_subsegment("hello") as subsegment:
+                subsegment.put_annotation("User", "unknown")
+                logger.info(f"Request from unknown received")
+                return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+        @logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST, log_event=True)
+        def lambda_handler(event, context):
+            global cold_start
+            with xray_recorder.in_subsegment("handler") as subsegment:
+                if cold_start:
+                    subsegment.put_annotation("ColdStart", "True")
+                    cold_start = False
+                else:
+                    subsegment.put_annotation("ColdStart", "False")
+                return app.resolve(event, context)
+    ```
+
+=== "template.yaml"
+
+    ```yaml hl_lines="15"
+    AWSTemplateFormatVersion: "2010-09-09"
+    Transform: AWS::Serverless-2016-10-31
+    Description: >
+        Sample SAM Template for powertools-quickstart
+    Globals:
+    Function:
+        Timeout: 3
+    Resources:
+        HelloWorldFunction:
+            Type: AWS::Serverless::Function
+            Properties:
+                CodeUri: hello_world/
+                Handler: app.lambda_handler
+                Runtime: python3.9
+                Tracing: Active
+                Events:
+                    HelloWorld:
+                    Type: Api
+                    Properties:
+                        Path: /hello
+                        Method: get
+                    HelloWorldName:
+                    Type: Api 
+                    Properties:
+                        Path: /hello/{name}
+                        Method: get
+    Outputs:
+        HelloWorldApi:
+            Description: "API Gateway endpoint URL for Prod stage for Hello World function"
+            Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
+    ```
+
+Few things happened here. Firstly, we imported required x-ray SDK classes. 
+`xray_recorder` is instance of global AWS X-Ray recorder class that will begin/end segments/subsegments and send them to the X-Ray daemon.
+In order to create new sub-segments we use `xray_recorder.in_subsegment` method as context manager. 
```

Comment:
Simplify:  ~~In order~~ to create a new

Reply:
Rewrote to:
To build new sub-segments, we use `xray_recorder.in_subsegment` method as a context manager. `